homeassistant/components/utility_meter/services.yaml

```diff
@@ -3,7 +3,9 @@
 reset:
   target:
     entity:
-      domain: select
+      - domain: select
+      - domain: sensor
+        integration: utility_meter
```

Comment:
This creates an UX bug where users now expect to be able to reset individual tariff sensors...

Reply:
I can certainly revert it to enable this functionality in YAML mode only. I don't use the tariff feature myself but is there a problem if they reset a tariff sensor directly?

Reply:
if will not work

Reply:
It's a UX aspect, reset service is used to reset all sensors at once (target is select), with this PR users can reset a single sensor and miss the remaining ones.

Why don't you setup a **single** tariff ? this way you could use reset without any change...

Reply:
Looking at the [documentation](https://www.home-assistant.io/integrations/utility_meter/) I already am running in single tariff mode.

> ### Supported tariffs
> 
> A list of supported tariffs, leave empty if only a single tariff is needed.

The utility meter is perfect for my needs except for being able to manually reset it.